src/pages/home/index.jsx

```diff
@@ -61,6 +73,24 @@ class Home extends Language {
               <Button type="primary" link={getLink(dataSource.brand.getStartedButton.link)}>{dataSource.brand.getStartedButton.text}</Button>
               <Button type="normal" link={getLink(dataSource.brand.viewOnGithubButton.link)}>{dataSource.brand.viewOnGithubButton.text}</Button>
             </div>
+            <div className="github-buttons">
+              <a href="https://github.com/apache/incubator-dubbo">
+                <div className="star">
+                  <img src="https://img.alicdn.com/tfs/TB1FlB1JwHqK1RjSZFPXXcwapXa-32-32.png" />
```

Comment:
Would you please inline this png file? Putting on a cdn may be unfriendly to foreign visitors.